modules/javafx.graphics/src/main/java/com/sun/javafx/font/Disposer.java

```diff
@@ -52,29 +52,18 @@ public class Disposer implements Runnable {
         disposerInstance = new Disposer();
 
         ThreadGroup tg = Thread.currentThread().getThreadGroup();
-        @SuppressWarnings("removal")
-        var dummy = java.security.AccessController.doPrivileged(
-            new java.security.PrivilegedAction() {
-                @Override
-                public Object run() {
-                    /* The thread must be a member of a thread group
-                     * which will not get GCed before VM exit.
-                     * Make its parent the top-level thread group.
-                     */
-                    ThreadGroup tg = Thread.currentThread().getThreadGroup();
-                    for (ThreadGroup tgn = tg;
-                         tgn != null;
-                         tg = tgn, tgn = tg.getParent());
-                    Thread t =
-                        new Thread(tg, disposerInstance, "Prism Font Disposer");
-                    t.setContextClassLoader(null);
-                    t.setDaemon(true);
-                    t.setPriority(Thread.MAX_PRIORITY);
-                    t.start();
-                    return null;
-                }
-            }
-        );
+        /* The thread must be a member of a thread group
+         * which will not get GCed before VM exit.
+         * Make its parent the top-level thread group.
+         */
+        for (ThreadGroup tgn = tg;
+            tgn != null;
+            tg = tgn, tgn = tg.getParent());
+        Thread t = new Thread(tg, disposerInstance, "Prism Font Disposer");
```

Comment:
very minor: I would have separated `for()` from L62 by a newline.  This `for` is already confusing enough.

Reply:
Done